test/e2e/framework/util.go

```diff
@@ -2461,12 +2463,17 @@ func WaitForAllNodesSchedulable(c clientset.Interface, timeout time.Duration) er
 		//
 		// However, we only allow non-ready nodes with some specific reasons.
 		if len(notSchedulable) > 0 {
-			Logf("Unschedulable nodes:")
-			for i := range notSchedulable {
-				Logf("-> %s Ready=%t Network=%t",
-					notSchedulable[i].Name,
-					IsNodeConditionSetAsExpected(notSchedulable[i], v1.NodeReady, true),
-					IsNodeConditionSetAsExpected(notSchedulable[i], v1.NodeNetworkUnavailable, false))
+			// In large clusters, log them only every 10th pass.
+			largeClusterThreshold := 100
```

Comment:
it might be useful in other places as well - maybe create a file-scoped constant?